clients/pricing-client/src/openapi.d.ts

```diff
@@ -353,6 +353,240 @@ declare namespace Components {
          * The checkout mode for the cart checkout.
          */
         export type CheckoutMode = "create_order" | "create_invoice" | "create_quote";
+        /**
+         * The price entity schema for dynamic pricing
+         * example:
+         * {
+         *   "$ref": "#/components/examples/composite-price"
+         * }
+         */
+        export interface CompositePrice {
+            [name: string]: any;
+            /**
+             * Whether the price can be used for new purchases.
+             */
+            active?: boolean;
+            /**
+             * A brief description of the price.
+             */
```

Comment:
We're missing the `currency` field here.

Reply:
On it.